noxfile.py

```diff
@@ -65,7 +65,14 @@ class = StreamHandler
         'account',
         'name',
         existing_type=sa.String(70),
-    )"""
+    )
+    op.alter_column(
+        'account',
+        'description',
+        existing_type=sa.Unicode(200),
+        nullable=False,
+    )
+    """
```

Comment:
These strings are used in migration tests as a migration script.
Adding an operation, which was failing in the original issue, to make sure the case is tested.